src/std/text.ab

```diff
@@ -159,16 +160,18 @@ pub fun capitalize(text: Text): Text {
 
 /// Pads `text` with the specified `pad` character on left until it reaches the desired `length`.
 pub fun lpad(text: Text, pad: Text, length: Num): Text {
-    if length <= len(text): return text
-    length = len(text) - length
+    if length <= len text: return text
+    length = len text
```

Comment:
I am not sure about split those in 2 lines but when the issue is gone I will check that part